services/utils/knack.py

```diff
@@ -17,6 +17,19 @@ def socrata_formatter_point(value):
     }
 
 
+def socrata_formatter_multipoint(value):
+    if not value:
+        return value
+
+    try:
+        return {
+            "type": "MultiPoint",
+            "coordinates": [[float(v["longitude"]), float(v["latitude"])] for v in value],
+        }
+    except ValueError:
```

Comment:
i like the try/catch here!

Reply:
@chiaberry would mind adding to this function a comment with a link to [this socrata doc page](https://dev.socrata.com/docs/datatypes/multipoint.html)?